.github/workflows/update-repo.yml

```
@@ -0,0 +1,44 @@
+name: Update plugin repo
+on:
+  schedule:
+    - cron: "0 */6 * * *"
+  workflow_dispatch:
+permissions:
+  contents: write
+jobs:
+  PluginRepo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Joplin plugin repository
+        uses: actions/checkout@v4
+        with:
+          ref: master
+          path: 'plugins'
+
+      - name: Install Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Prepare git
+        shell: bash
+        # Makes the commit author be shown as GitHub Actions in the
+        # GitHub UI.
+        # See https://github.com/actions/checkout/pull/1707
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Or we could use the github bot. I am not so sure about that. I also use sometimes the github action user for commits, but we do have a bot. On the other side it is a github action... Hmm, what to do? 🤔 

```suggestion
          git config --global user.name "joplinbot"
          git config --global user.email "bot@joplinapp.org"
```